### PR TITLE
[TRAFODION-3235] add div and trim function for map feature, improve bad file print function.

### PR DESCRIPTION
**Pull request by SuJinpei**
Some customer asks for this feature and these feature has been tested.

### CODE REVIEW

File: core/conn/odb/src/odb.c [hunk 1]
```diff
@@ -6246,7 +6248,8 @@ static void Oload(int eid)
              trt[16];           /* translit to array */         
         char op;                /* 1=substr, 2=dconv, 3=tconv, 4=tsconv, 5=replace,
                                    6=toupper, 7=tolower, 8=firstup, 9=csubstr, 10=translit,
-                                   11=comp, 12=comp3, 13=zoned, 14=emptyasconst, 15=emptyasempty */
+                                   11=comp, 12=comp3, 13=zoned, 14=emptyasconst, 15=emptyasempty,
+                                   16=div */
```
Comment (DaveBirdsall) [inline core/conn/odb/src/odb.c]: Should the comments also include 17=trim?

File: core/conn/odb/src/odb.c [hunk 2]
```diff
@@ -10076,6 +10114,7 @@ static int Oloadbuff(int eid)
         clock_gettime(CLOCK_MONOTONIC, &tsp1);
 #endif
         Or = SQLExecute(thps[tid].Os) ;         /* Execute INSERT (load/copy) or tgt command */
+        SQLLEN tLastRow = -1;           /* this is special solution for China Union Pay, print only first error message for state 22003 */
```
Comment (DaveBirdsall) [inline core/conn/odb/src/odb.c]: Probably best not to put customer names into the code. Consider changing the comment to just "/* print only first error message for state 22003 */"

File: core/conn/odb/src/odb.c [hunk 3]
```diff
@@ -14685,6 +14743,56 @@ static void addGlobalPointer(void *ptr)
...
+            else return 0;
+            break;
+        case 7:
+            if (!isdigit(str[i])) return 0;
+            break;
+        default:
+            return 0;
+        }
+    }
+    if (s == 3 || s == 7 || s == 5) return 1;
```
Comment (DaveBirdsall) [inline core/conn/odb/src/odb.c]: I am wondering if s == 4 should be allowed here too. Consider the string "1234."

File: core/conn/odb/src/odb.c (hunk 1 above)
Comment (SuJinpei) [reply core/conn/odb/src/odb.c]: Thank you, I will fix this.

File: core/conn/odb/src/odb.c (hunk 2 above)
Comment (SuJinpei) [reply core/conn/odb/src/odb.c]: Thank you, I will fix this.

File: core/conn/odb/src/odb.c (hunk 3 above)
Comment (SuJinpei) [reply core/conn/odb/src/odb.c]: Yes, this is also a valid numeric value.

**Pull request merged**